src/Symfony/Bundle/FrameworkBundle/Resources/config/web.xml

```diff
@@ -46,5 +46,9 @@
             <argument type="service" id="request_stack" />
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="validate_request_client_ip_listener" class="Symfony\Component\HttpKernel\EventListener\ValidateRequestClientIpListener">
```

Comment:
If we rename the class as I suggest, this should be renamed accordingly (`validate_request_listener`).